src/App.js

```diff
@@ -1,23 +1,27 @@
-import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import { observer } from 'mobx-react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
-import Container from '@containers/Container'
-import Splash from '@containers/Splash'
+import PrivateRoute from '@components/PrivateRoute';
+import Landing from '@containers/Landing'
 import Uploads from '@containers/Uploads'
+import Store from '@models/Store';
 
 import './App.css';
 
-function App() {
+const App = observer(() => {
```

Comment:
This makes the class so it will re-render if any of the values "observed" changes. 

In this case, the observed value is `Store` as indicated on the class itself.